experimental/dds/tree2/src/feature-libraries/editable-tree/editableField.ts

```diff
@@ -61,8 +61,19 @@ export function makeField(
 	fieldSchema: FieldStoredSchema,
 	cursor: ITreeSubscriptionCursor,
 ): EditableField {
-	const targetSequence = new FieldProxyTarget(context, fieldSchema, cursor);
-	return adaptWithProxy(targetSequence, fieldProxyHandler);
+	const fieldAnchor = cursor.buildFieldAnchor();
+
+	const targetSequence = new FieldProxyTarget(context, fieldSchema, cursor, fieldAnchor);
+	const output = adaptWithProxy(targetSequence, fieldProxyHandler);
+	if (fieldAnchor.parent !== undefined) {
```

Comment:
Since the behavior here might seems strange, lets clarify it:
```suggestion
	// Fields currently live as long as their parent does.
	// For root fields, this means forever, but other cases can be cleaned up when their parent anchor is deleted:
	if (fieldAnchor.parent !== undefined) {
```

Reply:
Added suggested clarifying comments